aws/amazon-freertos
# check service data in ADV

Description
-----------

 - add connection steps for "callback null check"
 - check service data in ADV
 - clear event queue before setproperty
 - add argument to enable/disable the check for included_service&secondary_service
 - add macro for iot_logging_setup.h

Checklist:
----------


- [ ] My code is Linted.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
